modules/core/04-channel/types/msgs_test.go

```diff
@@ -103,6 +109,16 @@ func (suite *TypesTestSuite) SetupTest() {
 	suite.proof = proof
 }
 
+// SetupTestCoordinator creates a coordinator with 2 test chains.
+func (suite *TypesTestSuite) SetupTestCoordinator() {
+	suite.coordinator = ibctesting.NewCoordinator(suite.T(), 2)
+	suite.chainA = suite.coordinator.GetChain(ibctesting.GetChainID(1))
+	suite.chainB = suite.coordinator.GetChain(ibctesting.GetChainID(2))
+	// commit some blocks so that QueryProof returns valid proof (cannot return valid query if height <= 1)
+	suite.coordinator.CommitNBlocks(suite.chainA, 2)
+	suite.coordinator.CommitNBlocks(suite.chainB, 2)
```

Comment:
I think this setup code is outdated. Could we try removing it and see if tests pass?